vaadinfrontend/src/main/java/life/qbic/datamanager/views/ContactElement.java

```diff
@@ -0,0 +1,55 @@
+package life.qbic.datamanager.views;
+
+import com.vaadin.flow.component.html.Label;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import life.qbic.projectmanagement.domain.project.PersonReference;
+
+import java.util.Objects;
+
+/**
+ * <b>Contact Element</b>
+ *
+ * <p>Element consisting of a vertical layout and labels to represent a {@link PersonReference} contact via full name and email.
+ * Furthermore, the element contains specific styling for the labels.
+ * </p>
+ * <p>Use this element when you want to represent contact data of a person!</p>
+ *
+ * @since 1.0.0
+ */
+public class ContactElement extends VerticalLayout {
+  private final Label projectManagerLabel;
```

Comment:
If this component is meant to display PersonReference objects, I think the field should not be called projectManager*

Reply:
How about `Label nameField` instead?